app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -66,6 +66,10 @@ $thumbnail-icon-border-colour: govuk-colour("mid-grey", $legacy: "grey-3");
   }
 }
 
+.gem-c-attachment__metadata--compact {
```

Comment:
Could you help me understand the need for this style? As I understand it, we can have up to three lines of metadata for an attachment (ref, unnumbered, file type). I would have thought this needs to apply to one of the other lines, as well, if it applies to any.

Reply:
As per design, the reference-related lines don't have margin-bottom, compared to the other metadata lines which require a margin-bottom – this is the reason I introduced this modifier, to be able to style them differently.

Reply:
Ahhh, sorry. I just realised this applies to **all** metadata lines. All good 👍